rtvi-client-js-daily/src/transport.ts

```diff
@@ -28,12 +28,8 @@ export interface DailyTransportAuthBundle {
 }
 
 export class DailyTransport extends Transport {
-  protected _state: TransportState = "idle";
```

Comment:
I have removed this variable from here, because it is already declared inside the abstract class `Transport`.